composer.json

```diff
@@ -4,8 +4,8 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "php": "^8.0",
-        "dvsa/mot-cpms-notifications": "^1.0.0",
+        "php": "^8.1",
+        "dvsa/mot-cpms-notifications": "dev-feature-BL-15591-update-cpms-php-81",
```

Comment:
Branch refs for dev branch?

Reply:
Once reviewed, I will create tags and sort it out before merging.